contracts/noop/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "noop"
+version = "0.1.0"
+authors = [
+    "Anatoly Yakovenko <anatoly@solana.com>",
+    "Greg Fitzgerald <greg@solana.com>",
+    "Stephen Akridge <stephen@solana.com>",
+    "Michael Vines <mvines@solana.com>",
+    "Rob Walker <rob@solana.com>",
+    "Pankaj Garg <pankaj@solana.com>",
+    "Tyera Eulberg <tyera@solana.com>",
+    "Jack May <jack@solana.com>",
+]
+
+[dependencies]
+libloading = "0.5.0"
+solana = { path = "../.." }
```

Comment:
This is rough. Would it be a huge amount of work to move the noop dependencies out of solana into their own workspace?

Reply:
The amount of work is not what I'm concerned about.  I would like to have example dynamic programs built into the solana project so they are part of the normal dev/test/ci/etc... cycle.

What is your concern with referencing solana in this way?

Reply:
The circular reference will make it so it's impossible to get `noop` out on its own.

Reply:
In or out noop will have to depend on Solana (or a subset) in order to share the common interface definitions